common/app/views/support/Profile.scala

```diff
@@ -21,7 +21,7 @@ sealed trait ElementProfile {
   def autoFormat: Boolean
 
   def elementFor(image: ImageMedia): Option[ImageAsset] = {
-    val sortedCrops = image.imageCrops.sortBy(-_.width)
```

Comment:
I don't remember that being intentional, and I also think that the logic predates the two incarnations of image resizing we've used, plus the introduction of master images. So I'd say choose an approach for image selection (whilst resizing is disabled) which the team thinks works for you today (and tomorrow).